app/controllers/keywords_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class KeywordsController < ApplicationController
+  before_action(only: %i[new create]) { login_required(t('keyword.page')) }
```

Comment:
`before_action(only: %i[new create]) { login_required(t('keyword.page')) }` this looks a bit weird, can we make it like the other place, like

```
before_action :xxx, only: %i[new create]
```

Reply:
Fixed in da5ab9367c98fb14b825fa29d049d1d06b2615db